tests/unit/config/config_test.py

```diff
@@ -346,6 +346,11 @@ def test_load_with_multiple_files_and_extends_in_override_file(self):
         tmpdir.join('common.yml').write("""
             base:
               labels: ['label=one']
+              ulimits:
+                nproc: 65535
+                memlock:
+                    soft: 1024
+                    hard: 2048
```

Comment:
I think this should be added as a separate case in the `ExtendsTest` TestCase. This test is unrelated